app/lib/json_web_token.rb

```diff
@@ -0,0 +1,27 @@
+class JsonWebToken
+  SECRET_KEY = Rails.application.secrets.secret_key_base.to_s
+
+  # Expects a hash {user_id: id}
+  def self.encode(payload, exp = 24.hours.from_now)
+    payload[:exp] = exp.to_i
+    begin 
+      JWT.encode(payload, SECRET_KEY)
+    rescue JWT::EncodeError
+      render json: {}, status: 401 # Unauthorized 
```

Comment:
is `render` in scope here? I had all my recuses in the authorize_request because I know the `render` method is in scope their. That's the only reason i had that triple stack rescue.